coralproject/talk
# UX Comment Component

## What does this PR do?
- Adds designs for the Comment Component

## How do I test this PR?
- Go to `/admin/moderate`
- Check the modqueues
- They should match the style
- Check that everything is working as expected

https://www.pivotaltracker.com/story/show/139964323
